docs/02-install.md

```diff
@@ -237,6 +241,6 @@ the JSON and Terraform state files. If you have any desire to retain the
 original `intsall-config.yaml` file, be sure to make a copy.
 
 # Problems?
-If you had installation issues, see the [troubleshooting](06-troubleshooting.md) section.
+If you had installation issues, see the [troubleshooting](07-troubleshooting.md) section.
```

Comment:
Typo here, https://github.com/openshift/training/pull/437/files#diff-04c6e90faac2675aa89e2176d2eec7d8R30
How about rename it to **99-troubleshooting.md**, so we do not need to change it again and again

Reply:
Fixed with c956ae2